src/relevance_feedback/relevance_feedback.py

```diff
@@ -0,0 +1,81 @@
+import cv2, os
```

Comment:
Any reason to have separate folder for relevance_feedback? I am of the opinion to keep all backend related files in src/backend folder

Reply:
Yes let's keep everything in the same folder